smalltalksrc/VMMaker/SpurMemoryManager.class.st

```diff
@@ -9357,10 +9357,13 @@ SpurMemoryManager >> newExternalAddressWithValue: aPointer [
 
 { #category : #'header access' }
 SpurMemoryManager >> newHashBitsOf: objOop [
+
+	<api>
+	<option: #SpurObjectMemory>
```

Comment:
wo, this was not there?!

Reply:
Nop, because the trampoline in normal JIT is calling `ceNewHashOf:` (with the integer tagging).

I'm calling this because the code of the tagging is already in the interpreter.
I could change the interpreter also I think now...

Reply:
HAha, ok, this was fun